### DISCUSSION TITLE
Multilingual (AI Translations)

### DISCUSSION THREAD

**Discussion by curtissrickard** [webstudio-is/webstudio-community#79] [Ideas]
Would be great if we could translate created pages to whatever language in one go.

**Comment by kof** [comment]
Agree, AI translations are easy to implement.

**Reply by kwameandco** [reply]
Would it be something like Webflow Locales?
That would be incredible as I'm seeing more and more clients think about this kind of implementation.
I would be really keen on WebStudio localisation even without AI.
